spyderlib/spyder.py

```diff
@@ -358,7 +358,14 @@ def __init__(self, options=None):
         # Tour  # TODO: Should I consider it a plugin?? or?
         self.tour = None
         self.tours_available = None
-        
+
+        # Check for updates Thread and Worker, refereces needed to prevent
+        # segfaulting
+        self.check_updates_action = None
+        self.thread_updates = None
+        self.worker_updates = None
+        self.flag_updates_feedback = True

```

Comment:
Sorry, last thing :-) I don't like too much this variable name.

Suggestions:
1. `self.updates_feedback`
2. `self.give_updates_feedback`

I think any of those would convey its meaning better ;-)